Makefile

```diff
@@ -87,36 +87,30 @@ proto: install-tools ## Generate protobuf files
 cleanup-warehouse-integration:
 	docker-compose -f warehouse/docker-compose.test.yml down --remove-orphans --volumes
 
+define generate_namespace
+$(shell echo $(1)-$(shell shuf -i 1-1000000 -n 1)-$(shell date +%s))
+endef
+
 setup-warehouse-integration: cleanup-warehouse-integration
-	BIGQUERY_INTEGRATION_TEST_SCHEMA=bwh-$(shell uuidgen)	\
-	REDSHIFT_INTEGRATION_TEST_SCHEMA=rwh-$(shell uuidgen)	\
-	SNOWFLAKE_INTEGRATION_TEST_SCHEMA=swh-$(shell uuidgen) 	\
-	DATABRICKS_INTEGRATION_TEST_SCHEMA=dwh-$(shell uuidgen) 	\
+	BIGQUERY_INTEGRATION_TEST_SCHEMA=$(call generate_namespace, bwh) \
+	REDSHIFT_INTEGRATION_TEST_SCHEMA=$(call generate_namespace, rwh) \
+	SNOWFLAKE_INTEGRATION_TEST_SCHEMA=$(call generate_namespace, swh) \
+	DATABRICKS_INTEGRATION_TEST_SCHEMA=$(call generate_namespace, dwh) \
 	docker-compose -f warehouse/docker-compose.test.yml up --build start_warehouse_integration
 
 logs-warehouse-integration:
 	docker logs wh-backend
 
 run-warehouse-integration: setup-warehouse-integration
-	if docker-compose -f warehouse/docker-compose.test.yml exec -T wh-backend go test -v ./warehouse/... -tags=warehouse_integration -p 8 -timeout 30m -count 1; then \
+	$(eval TEST_PATTERN = '^Test.*Integration$$$$$$$$' '^Test.*ConfigurationValidation$$$$$$$$')
```

Comment:
Do we need multiple `$` here? 

Also, should Integration be a prefix or postfix:

`TestIntegratioBigQuery` vs `TestBigQueryIntegration`

The second is more readable, but the first more convenient due to the hierarchy

Reply:
> Do we need multiple $ here?

Actually, we are using it in 2 eval expressions and then we are running the command.  Basically, we need to escape it 3 times.
2^3 = 8

Reply:
Yes, in terms of hierarchy `TestIntegratioBigQuery` is the best choice.